core/src/main/scala/kafka/security/auth/Authorizer.scala

```diff
@@ -18,7 +18,7 @@
 package kafka.security.auth
 
 import kafka.network.RequestChannel.Session
-import org.apache.kafka.common.Configurable
+import org.apache.kafka.common.{KafkaException, Configurable}
```

Comment:
There was a throws clause, but I removed that, so not anymore. Removed it locally.